app/assets/javascripts/gobierto_budgets/maps.js

```diff
@@ -202,67 +62,438 @@ $(document).on('turbolinks:load', function() {
         unit: ' ' + I18n.t('gobierto_budgets.pages.map.people'),
+      },
+      ingreso_habitante: {
+        name: I18n.t('gobierto_budgets.pages.map.income_per_inhabitant')
       }
     };
 
-    cartodb.createVis('map', 'https://gobierto.carto.com/api/v2/viz/205616b2-b893-11e6-b070-0e233c30368f/viz.json', {
-      shareable: false,
-      title: false,
-      description: false,
-      search: false,
-      tiles_loader: true,
-      center_lat: window.mapSettings.centerLat,
-      center_lon: window.mapSettings.centerLon,
-      zoom: window.mapSettings.zoomLevel,
-      zoomControl: true,
-      loaderControl: true
-    })
-    .done(function(vis, layers) {
-      var sublayer = layers[1].getSubLayer(0);
-      vis.addOverlay({
-        type: 'tooltip',
-        layer: sublayer,
-        template: $('#infowindow_template').html(),
-        position: 'bottom|right',
-        fields: [{ name: 'name', value: 'value', valuef: 'valuef', indicator_name: 'indicator_name', unit: 'unit', place_id: 'place_id' }]
-      });
-      sublayer.setInteractivity('name, value,valuef,indicator_name,unit,place_id');
-      renderMapIndicator(sublayer, vis);
-      renderMapBudgetLine(sublayer, vis);
-      $('[data-indicator].selected').click();
-
-      // On load, hide the overlay and reset the tile spinner
-      layers[1].on("load", function() {
-        $('#map .overlay').css({
-          'display': 'none'
+    var completeIndicator = indicatorsValue.gasto_por_habitante.unit
+    var tooltipString = indicatorsValue.gasto_por_habitante.name
+
+    indicators.forEach(
+      function(indicator) {
+        indicator.addEventListener("click", loadIndicators);
+      }
+    );
+
+    var CHOROPLET_SCALE = [
+      [255, 255, 201],
+      [192, 229, 174],
+      [117, 198, 179],
+      [59, 173, 187],
+      [30, 134, 181],
+      [31, 83, 155],
+      [14, 39, 118]
+    ]
+
+    var INITIAL_VIEW_STATE = {
+      latitude: 40.416775,
+      longitude: -3.703790,
+      zoom: 5,
+      minZoom: 5,
+      maxZoom: 8
+    };
+
+    deckgl = new deck.Deck({
+      canvas: 'map',
+      initialViewState: INITIAL_VIEW_STATE,
+      controller: true,
+      getTooltip: getTooltip,
+      onClick: onClick,
+      onLoad: onLoad,
+      onHover: function onHover(_ref2) {
+        var object = _ref2.object;
+        return isHovering = Boolean(object);
+      },
+      getCursor: function getCursor(_ref3) {
+        var isDragging = _ref3.isDragging;
+        return isDragging ? 'grabbing' : isHovering ? 'pointer' : 'grab';
+      },
+      onViewStateChange: function onViewStateChange(_ref) {
+        var viewState = _ref.viewState;
+        return deckgl.setProps({
+          viewState: viewState
+        });
+      }
+    });
+
+
+    function onClick(info, event) {
+      var municipality = info.object.properties.name
+      var replaced = municipality.replace(/ /g, '-');
+      window.location.href = "/places/" + accentsTidy(replaced) + "/" + year;
```

Comment:
This function is broken, i.e http://presupuestos.gobify.net/places/gallues/galoze/2020

You should use the `slug` attribute from municipalities table, which already removes accents and so on.